### PR TITLE
🚚 Rename Linux package & add AppImage support

### PR DESCRIPTION
**Pull request by prateekmedia**
Appimage currently doesn't include all the required libraries, so this PR is draft.

### CODE REVIEW

**Comment by prateekmedia** [comment]
@alexmercerind I have also submitted a [draft PR to flathub](https://github.com/flathub/flathub/pull/2507), which would get approved after this is merged and some issues are resolved in it.

**Comment by alexmercerind** [comment]
Thanks a lot for your efforts.

File: README.md [hunk 1]
```diff
@@ -19,14 +19,15 @@
...
 
-- [Setup Installer](https://github.com/harmonoid/harmonoid/releases/download/v0.1.3/harmonoid-windows-setup.exe) (13.2 MB) _Recommended. Automatically links with various file extensions_
-- [Portable EXE](https://github.com/harmonoid/harmonoid/releases/download/v0.1.3/harmonoid-windows-exe.zip) (15.7 MB)
+- [Setup Installer](https://github.com/harmonoid/harmonoid/releases/download/latest/harmonoid-windows-setup.exe) (13.2 MB) _Recommended. Automatically links with various file extensions_
+- [Portable EXE](https://github.com/harmonoid/harmonoid/releases/download/latest/harmonoid-windows-exe.zip) (15.7 MB)
 
 #### **Linux**
 
-- [DEB Package](https://github.com/harmonoid/harmonoid/releases/download/v0.1.3/harmonoid-linux-debian.deb) (9.96 MB) _e.g. Ubuntu, Debian, elementaryOS, KDE Neon etc._
+- [DEB Package](https://github.com/harmonoid/harmonoid/releases/download/latest/harmonoid-linux-debian.deb) (9.96 MB) _e.g. Ubuntu, Debian, elementaryOS, KDE Neon etc._
```
Comment (alexmercerind) [inline README.md:27]: `download/latest` must be `latest/download`

**Comment by prateekmedia** [comment]
You can currently merge this, but the appimage doesn't contain any statically linked `libmediainfo` or `gstreamer-libav` or `webkit2gtk`.

**Pull request merged**